pp3/module/Application/src/Application/Controller/AdminController.php

```diff
@@ -350,32 +350,37 @@ public function emailSendingAction() {
 
         $onlyVerified = false;
         $nbVersionId = [];
-        $subject = '[NetBeans PluginPortal] <SUBJECT>';
+        $subject = '[NetBeans PluginPortal] Check your plugin with the new Apache NetBeans %4$s version!';
         $emailText = '
 <html>
 <head><title></title></head>
 <body>
-<p>Dear %1$s,</p>
+<p>Hey %1$s,</p>
 
-<p>we have some news for you.</p>
-
-<p>&lt;Your message&gt;</p>
+<p>
+have you heard of the new Apache NetBeans version on the horizon?
+Yes, the NetBeans community is going to release version %4$s soon!
+In order to have your great plugin(s) available on the NetBeans Update
+Center also for users of the new version please consider testing the
+plugin(s) with the latest RC build and possibly submit new verification
+request(s) via the Plugin Portal website.
+</p>
 
 <p>
 Your plugins:<br />
 %3$s
 </p>
 
-<p>Direct link to the plugin portal:<br />
-<a href="%2$s">%2$s</a>
+<p>Direct link to the list of your plugins:<br />
+<a href="%2$s/plugin/list">%2$s/plugin/list</a>
```

Comment:
This will fail in most cases. The URL `/plugin/list` will redirect you to the front page because you are not logged in. This might confuse users reading the sentence directly before the link.

Reply:
You are right but I would not underestimate plugin developers.  For those already logged in the link will work and the others will quickly figure out what happened and navigate to My Plugins view.

Or are you suggesting to just provide the link to Plugin Portal homepage?

Reply:
I can live with it. My perspective was, that at most time I would hit the redirect case, not the "direct to plugin list" case. But I have no strong feelings here.

Reply:
Thank you Matthias. In such case I would like to merge this so that you can redeploy the Plugin Portal sources ASAP in order to inform all plugin owners about NB17 RC1. Please approve if you agree.